gemsurance.gemspec

```diff
@@ -19,12 +19,12 @@ Gem::Specification.new do |s|
   s.files                     = `git ls-files -- bin lib`.split("\n")
   s.executables               = ["gemsurance"]
 
-  s.add_dependency("bundler", "~> 1.2")
+  s.add_dependency("bundler", ">= 1.2")
   s.add_dependency("git", "~> 1.2")
   s.add_dependency("gems", ">= 0.8")
 
-  s.add_development_dependency("mocha", "0.14.0")
-  s.add_development_dependency("rake", "0.9.2.2")
-  s.add_development_dependency("nokogiri", "1.10.0")
-  s.add_development_dependency("test-unit", "3.0.9")
+  s.add_development_dependency("mocha", "1.9.0")
+  s.add_development_dependency("rake", ">= 10.0.0")
```

Comment:
I think this should be OK. I doubt we make use of any old rake features.
Although note what's in [gem_info_retriever_test.rb](https://github.com/appfolio/gemsurance/tree/master/test/unit/gem_info_retriever_test.rb)